app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -100,11 +100,13 @@ class Config {
       'and make sure this patch meets those guidelines before LGTMing.';
 
   String get goldenTriageMessage => 'Nice merge! 🎉\n'
-      'It looks like this PR made changes to golden files. Be sure to visit '
+      'It looks like this PR made changes to golden files. If these changes have'
+      ' not been triaged as a tryjob, be sure to visit '
```

Comment:
nit: moves this space to the end of the previous line (to be symmetric with the other lines).